ui/views_test.py

```diff
@@ -269,7 +269,7 @@ def test_collection_viewset_permissions(logged_in_apiclient):
         reverse('models-api:collection-detail', kwargs={'key': uuid4().hex}),
     )
     for url in urls:
-        assert client.get(url).status_code == status.HTTP_403_FORBIDDEN
+        assert client.get(url).status_code == status.HTTP_200_OK
```

Comment:
This test fails with a 404 because it is looking for a collection that doesn't exist, try this:

```python
def test_collection_viewset_permissions(logged_in_apiclient):
    """
    Tests the list of collections for an anonymous_user
    """
    client, _ = logged_in_apiclient
    client.logout()
    collection = CollectionFactory()
    urls = (
        reverse('models-api:collection-list'),
        reverse('models-api:collection-detail', kwargs={'key': collection.hexkey}),
    )
    for url in urls:
        assert client.get(url).status_code == status.HTTP_200_OK
        assert client.post(url, {'owner': 1}).status_code == status.HTTP_403_FORBIDDEN
```